src/vm/inst_sound.cpp

```diff
@@ -44,18 +44,24 @@ VmInstruction(InstSEplay) {
   StartInstruction;
   PopUint8(channel);
   PopUint8(type);
+
   if (type != 2) {
     PopExpression(effect);
     PopExpression(loop);
     ScrWork[SW_SEREQNO + channel] = effect;
     Audio::Channels[Audio::AC_SE0 + channel]->Volume =
         (ScrWork[SW_SEVOL + channel] / 100.0f) * 0.3f;
+
+    // Hack to prevent playing the first sound byte before pausing
```

Comment:
When pausing playback immediately after starting it, like this code does. A few milliseconds of sound is still played before it can be paused. This workaround is not a good solution, but I think we might have to extend the `AudioChannel` interface in order to support this use case.